src/main/java/com/gibraltar/iberia/items/ItemPersonalCompass.java

```diff
@@ -8,46 +8,36 @@
  */
 package com.gibraltar.iberia.items;
 
-import net.minecraft.client.renderer.block.model.ModelResourceLocation;
 import net.minecraft.creativetab.CreativeTabs;
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.EntityLivingBase;
 import net.minecraft.entity.item.EntityItemFrame;
-import net.minecraft.entity.player.EntityPlayer;
-import net.minecraft.init.Items;
 import net.minecraft.item.IItemPropertyGetter;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.NBTTagCompound;
-import net.minecraft.util.ActionResult;
-import net.minecraft.util.EnumActionResult;
-import net.minecraft.util.EnumHand;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.math.MathHelper;
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.world.World;
-import net.minecraftforge.client.model.ModelLoader;
 import net.minecraftforge.fml.common.FMLLog;
-import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.fml.common.registry.ForgeRegistries;
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-import java.lang.Thread;
-import java.util.List;
 import java.util.HashMap;
 import javax.annotation.Nullable;
 
-import com.gibraltar.iberia.challenge.NavigationChallenge;
 import com.gibraltar.iberia.network.MessageRegistry;
 import com.gibraltar.iberia.network.MessageGetPlayerSpawn;
 import com.gibraltar.iberia.Reference;
 
 public class ItemPersonalCompass extends Item {
```

Comment:
Personal compasses have the wrong name in the tooltips (i.e. not loading from the resources)

Reply:
lang files now enforce lowercase in 1.12. forgot to do that